NBitcoin.Tests/ProtocolTests.cs

```diff
@@ -262,11 +261,13 @@ public void CanProcessAddressGossip()
 
 				var nodeClient = node.CreateNodeClient();
 				nodeClient.VersionHandshake();
-				using (var list = nodeClient.CreateListener().Where(m => m.Message.Payload is AddrPayload))
+				AddrV2Payload addr;

```

Comment:
`var nodeClient = node.CreateNodeClient();` seems to be disposable too.

Reply:
Done